docs/mkdocs/docs/development/quick-start.md

```
@@ -0,0 +1,205 @@
+
+!!! info
+
+    Poetry is a tool for dependency management and packaging in Python. It allows you to declare the libraries your
+    project depends on and it will manage (install/update) them for you. Poetry offers a lockfile to ensure repeatable
+    installs, and can build your project for distribution.
+
+Once the Dev Container is started, open a shell session in the Terminal view of VS Code. The current working directory
+will be `/workspaces/partcad` containing the source files. To install Python packages, run the following:
+
+```bash
+$ poetry install
```

Markdownlint (0.37.0)

Dollar signs used before commands without showing output

(MD014, commands-show-output)